wallet/wallet-cli-lib/src/config.rs

```diff
@@ -135,6 +135,22 @@ pub struct CliArgs {
     /// Enable running the wallet service without RPC authentication (assuming wallet RPC server is enabled)
     #[arg(long, conflicts_with_all(["wallet_rpc_password", "wallet_rpc_username", "wallet_rpc_cookie_file"]))]
     pub wallet_rpc_no_authentication: bool,
+
+    /// Optional RPC address
```

Comment:
"Optionally, the wallet CLI can only be an interface to another remote wallet, accessible through RPC. So, you can start an RPC daemon, and make this CLI connect to it to control it. This is useful for servers, where the RPC wallet can be left staking, and the wallet CLI is used to control its state."

Reply:
replaced